alicloud/data_source_alicloud_db_instances.go

```diff
@@ -195,6 +195,75 @@ func dataSourceAlicloudDBInstances() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+
+						"ssl_expire_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"require_update": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"acl": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"ca_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"client_ca_cert": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"client_ca_cert_expire_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"client_cert_revocation_list": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"last_modify_status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"modify_status_reason": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"replication_acl": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"require_update_item": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"require_update_reason": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"ssl_create_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"ssl_enabled": {
+							Type:     schema.TypeString,
```

Comment:
Please ensure the `ssl_enabled` type whether is string.

Reply:
yes, it's string